### PR TITLE
Add the ability for area manager to close timeslots

### PR DESCRIPTION
**Pull request by vandahm**
Cory asked for the ability to close parts of the shop for classes. This PR does this.

What Cory does not yet realize is that, without the ability to reserve timeslots, the people she wishes to participate in those classes will have no way to get in the door. We'll have to save that for another PR.

### CODE REVIEW

File: accounts/utils.py [hunk 1]
```diff
@@ -1,13 +0,0 @@
-from .models import User
-
-def create_cognito_accounts():
-    for user in User.objects.exclude(email=''):
-        if user.email == 'ginamlarson@gmail.com':
```
Comment (automatingerror) [inline accounts/utils.py:5]: Userdata in code=not cool Kyle

**Review by automatingerror — COMMENTED** [review]
My mistake this was a removal not an addition.

**Comment by automatingerror** [comment]
> 
> 
> Cory asked for the ability to close parts of the shop for classes. This PR does this.
> 
> What Cory does not yet realize is that, without the ability to reserve timeslots, the people she wishes to participate in those classes will have no way to get in the door. We'll have to save that for another PR.





This links/relates back a similar issue during normal operations and the problem of activating keys for teachers and student who are not members.

File: accounts/utils.py (hunk 1 above)
Comment (vandahm) [reply accounts/utils.py:5]: That was committed by mistake. I'm removing it

**Pull request merged**